app/components/works/globus_setup_component.rb

```diff
@@ -10,7 +10,7 @@ def initialize(work_version:)
     attr_reader :work_version
 
     def render?
-      work_version.globus?
+      work_version.globus? && work_version.globus_endpoint
```

Comment:
Wondering about adding a `.present?` here. This field defaults to `nil` in the DB, I assume, and thus any non-nil value is fine here? We can't imagine this value being an empty string, right? If so, this is probably fine as is.